### PR TITLE
Bring up to date w/ previous work

### PR DESCRIPTION
**Pull request by mbarnett**

### CODE REVIEW

File: config/initializers/curation_concerns_actors.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CurationConcerns::Actors::ActorFactory.remove_actor CurationConcerns::Actors::AddToCollectionActor
```
Comment (murny) [inline config/initializers/curation_concerns_actors.rb:1]: This is breaking. These ActorFactory methods aren't defined anywhere?
https://travis-ci.org/ualbertalib/CollectionNorth

Comment (mbarnett) [reply config/initializers/curation_concerns_actors.rb:1]: Yeah, it's on master over in https://github.com/ualbertalib/hydranorth2. I just dumped the contents of a bunch of branches into master so I can plan out what needs to happen next sprint. Sorting out this cross-repo testing is part of it.

Comment (murny) [reply config/initializers/curation_concerns_actors.rb:1]: HN2 master is also broken, reason why i commented in here due to:
`Gem Load Error is: uninitialized constant CollectionNorth`
¯\_(ツ)_/¯

Comment (mbarnett) [reply config/initializers/curation_concerns_actors.rb:1]: Yes.

I don't really care that master is broken right now honestly. I just want a common starting point for planning, which I now have.

**Pull request merged**